src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -224,6 +227,57 @@ export const SearchResults = ({
         {searchResults}
       </ul>
 
+      {/*  */}
+      <div
+        className="pdf-alert-modal"
+        style={{
+          height: '500px',
+        }}
+      >
+        <Modal
+          onClose={() => toggleModalState()}
+          secondaryButton={{
+            action: () => {
+              toggleModalState();
+            },
+            text: 'Close',
+          }}
+          title="Adobe Reader DC Required"
+          visible={isOpen}
+        >
+          <>
+            <p className="vads-u-display--block vads-u-margin-bottom--3">
+              <span>
+                All PDF forms do not function fully in a web browser or other
+                PDF viewer. Please download the form and use Adobe Acrobat
+                Reader DC to fill out. For specific instructions about working
+                with PDF’s
```

Comment:
Should this be `PDFs`?

Reply:
No idea, I got it from content.

Reply:
I checked back with them, looks like I can change it, thanks for catching that.